src/os/bluestore/BlueStore.h

```diff
@@ -549,6 +549,12 @@ class BlueStore : public ObjectStore,
     bufferlist inline_bl;    ///< cached encoded map, if unsharded; empty=>dirty
 
     ExtentMap(Onode *o);
+    ~ExtentMap() {
+      auto p = extent_map.begin();
```

Comment:
you may use something like that instead:
extent_map.clear_and_dispose( [](Extent* e) {delete e;});